src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_params.py

```diff
@@ -126,48 +110,33 @@ def load_arguments(self, _):
-        c.argument('vault_name', vault_name_type, id_part='name')
-
-    with self.argument_context('backup azure-wl policy list') as c:
-        c.argument('vault_name', vault_name_type, id_part=None)
         c.argument('workload_type', wl_type)
+        c.argument('container_type', backup_management_type)
 
-    with self.argument_context('backup azure-wl policy show') as c:
-        c.argument('name', policy_name_type, options_list=['--name', '-n'], help='Name of the backup policy. You can use the backup policy list command to get the name of a policy.')
+    with self.argument_context('backup policy show') as c:
+        c.argument('container_type', backup_management_type)
 
     # Recovery Point
     # TODO: Need to use item.id once https://github.com/Azure/msrestazure-for-python/issues/80 is fixed.
     with self.argument_context('backup recoverypoint') as c:
         c.argument('vault_name', vault_name_type, id_part='name')
         c.argument('container_name', container_name_type)
         c.argument('item_name', item_name_type)
+        c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
+        c.argument('end_date', type=datetime_type, help='The end date of the range in UTC (d-m-Y).')
 
     with self.argument_context('backup recoverypoint list') as c:
         c.argument('vault_name', vault_name_type, id_part=None)
-        c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
-        c.argument('end_date', type=datetime_type, help='The end date of the range in UTC (d-m-Y).')
+        c.argument('workload_type', wl_type)
+        c.argument('container_type', backup_management_type)
 
     with self.argument_context('backup recoverypoint show') as c:
         c.argument('name', rp_name_type, options_list=['--name', '-n'], help='Name of the recovery point. You can use the backup recovery point list command to get the name of a backed up item.')
 
-    with self.argument_context('backup azure-wl recoverypoint') as c:
-        c.argument('vault_name', vault_name_type, id_part='name')
-        c.argument('container_name', container_name_type)
-        c.argument('item_name', item_name_type)
-        c.argument('workload_type', wl_type)
-
-    with self.argument_context('backup azure-wl recoverypoint list') as c:
-        c.argument('vault_name', vault_name_type, id_part=None)
-        c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
-        c.argument('end_date', type=datetime_type, help='The end date of the range in UTC (d-m-Y).')
-
-    with self.argument_context('backup azure-wl recoverypoint logchain show') as c:
-        c.argument('item_name', item_name_type, options_list=['--item-name', '-i'])
+    with self.argument_context('backup recoverypoint logchain show') as c:
         c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
```

Comment:
you can remote start_date and end_date as they are already mention in backup recoverypoint group. Line 125 and 126

Reply:
Yeah will do